[BUG] Flake8 pre-push test failure

### Software versions

Python version      :  3.10.4 | packaged by conda-forge | (main, Mar 24 2022, 17:43:32) [Clang 12.0.1 ]
IPython version     :  8.2.0
Tornado version     :  6.1
Bokeh version       :  3.0.0dev5+26.g7b039ec37
BokehJS static path :  /Users/jadalilleboe/Desktop/bokeh/bokeh/server/static
node.js version     :  v16.14.2
npm version         :  8.7.0
Operating system    :  macOS-12.1-x86_64-i386-64bit

### Browser name and version

_No response_

### Jupyter notebook / Jupyter Lab version

_No response_

### Expected behavior

I've just recently set up Bokeh locally and tried to add some example metadata (#11765). I expected the push to happen succesfully after fixing all the issues with the metadata.

### Observed behavior

I saw the error below when trying to push my changes.

### Example code

```Python
What I added as example metadata to examples/models/file/choropleth.py
'''This example shows an unemployment map by county of the United States of America in 2009.
It demonstrates the use of models such as a color bar, linear color mapper, and patches.

.. bokeh-example-metadata::
    :sampledata: us_counties, us_states, unemployment
    :apis: bokeh.models.ColorBar, bokeh.models.ColumnDataSource, bokeh.models.LinearColorMapper, bokeh.models.Patches, bokeh.models.Plot, bokeh.palettes.Viridis11, bokeh.embed.file_html, bokeh.document.document, bokeh.resources.INLINE, bokeh.transform.transform, bokeh.util.browser.view # noqa: E501
    :refs: :ref:`userguide_plotting` > :ref:`userguide_plotting_patches`
    :keywords: colormap, shapes, vector, polygon, colorbar, patches
'''
```


### Stack traceback or browser console output

```
dir = 'typings'

    def flake8(dir: str) -> None:
        ''' Assures that the Python codebase passes configured Flake8 checks.
    
        '''
        chdir(TOP_PATH)
        proc = run(["flake8", *ls_files(f"{dir}/**.py", f"{dir}/**.pyi")], capture_output=True)
>       assert proc.returncode == 0, f"Flake8 issues:\n{proc.stdout.decode('utf-8')}"
E       AssertionError: Flake8 issues:
E         typings/selenium/webdriver/common/keys.pyi:2:1: Y037 Use PEP 604 union types instead of typing.Union (e.g. "int | str" instead of "Union[int, str]").
E         
E       assert 1 == 0
E        +  where 1 = CompletedProcess(args=['flake8', 'typings/IPython/__init__.pyi', 'typings/IPython/core/__init__.pyi', 'typings/IPython...: Y037 Use PEP 604 union types instead of typing.Union (e.g. "int | str" instead of "Union[int, str]").\n', stderr=b'').returncode

tests/codebase/test_flake8.py:59: AssertionError
```

### Screenshots

<img width="883" alt="Screen Shot 2022-04-18 at 9 16 47 PM" src="https://user-images.githubusercontent.com/82007190/164040390-499e27d1-95ed-43d3-be32-f043f983c0fa.png">

--- a/typings/flaky.pyi
+++ b/typings/flaky.pyi
@@ -1,4 +1,5 @@
-from typing import Any, Callable, TypeVar
+from collections.abc import Callable
+from typing import Any, TypeVar
 
 from typing_extensions import TypeAlias
 

--- a/typings/ipykernel/comm/comm.pyi
+++ b/typings/ipykernel/comm/comm.pyi
@@ -1,4 +1,5 @@
-from typing import Any, Sequence
+from collections.abc import Sequence
+from typing import Any
 
 class Comm:
     def __init__(self, target_name: str = ..., data: Any = ..., metadata: dict[str, Any] | None = ..., buffers: Sequence[bytes] | None = ...) -> None: ...

--- a/typings/selenium/webdriver/common/keys.pyi
+++ b/typings/selenium/webdriver/common/keys.pyi
@@ -1,5 +1,5 @@
 # Standard library imports
-from typing import Sequence
+from collections.abc import Sequence
 
 from typing_extensions import TypeAlias
 

--- a/typings/selenium/webdriver/common/keys.pyi
+++ b/typings/selenium/webdriver/common/keys.pyi
@@ -1,9 +1,9 @@
 # Standard library imports
-from typing import Sequence, Union
+from typing import Sequence
 
 from typing_extensions import TypeAlias
 
-_KeySeq: TypeAlias = Union[str, int, Sequence[str]]
+_KeySeq: TypeAlias = str | int | Sequence[str]
 
 class Keys:
     NULL: str

--- a/typings/selenium/webdriver/support/wait.pyi
+++ b/typings/selenium/webdriver/support/wait.pyi
@@ -1,4 +1,5 @@
-from typing import Callable, Sequence, TypeVar
+from collections.abc import Callable, Sequence
+from typing import TypeVar
 
 from ..remote.webdriver import WebDriver
 

